### DISCUSSION TITLE
What happens to the plot if I shut down machinaris?

### DISCUSSION THREAD

**Discussion by FrancescoSaverioZuppichini** [guydavis/machinaris#217] [Q&A] [answered]
Hi guys,

I hope you are doing great. Assuming Machinaris is plotting and I restart it, what happens to the plots he was doing before? 

Thanks :) 

Francesco

**Comment by guydavis** [comment, marked as answer]
Good day!  Please see the [wiki](https://github.com/guydavis/machinaris/wiki/FAQ#can-i-restartupdate-the-container-without-killing-my-plotting-jobs).  If this answers your question, please mark this Discussion as Answered.
